community-operators/openshift-qiskit-operator/0.2.0/openshift-qiskit-operator.v0.2.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,331 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "qiskit.ibm.com/v1alpha1",
+          "kind": "QiskitPlayground",
+          "metadata": {
+            "name": "qiskitplayground-sample"
+          },
+          "spec": {
+            "image": "qiskit/jupyter:0.1",
+            "imagePullPolicy": "Always",
+            "pvc": "claim-admin",
+            "resources": {
+              "requests": {
+                "memory": "2G",
+                "cpu": "1"
+              },
+              "limits": {
+                "memory": "3G",
+                "cpu": "1.5"
+              }
+            }
+          }
+        }
+      ]
+    capabilities: Basic Install
+    categories: Developer Tools
+    certified: "false"
+    createdAt: 2021-004-23 22:25:53
+    containerImage: https://hub.docker.com/r/qiskit/operator
+    support: https://github.com/qiskit-community/openshift-quantum-operators/issues
+    repository: https://github.com/qiskit-community/openshift-quantum-operators/tree/master/operators-examples/kubernetes-qiskit-operator
+    description: |-
+      Launches a developement environment Jupyter notebok development environment for
+      developing quantum applications using Qiskit.
+    operators.operatorframework.io/builder: operator-sdk-v1.5.0
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
+  name: kubernetes-qiskit-operator.v0.2.0
```

Comment:
```suggestion
  name: openshift-qiskit-operator.v0.2.0
```

Reply:
Please remame it to correct package name

Reply:
Done